tokio-io/src/async_write.rs

```diff
@@ -82,7 +82,7 @@ pub trait AsyncWrite: std_io::Write {
     /// appropriate. This method is the hook for such protocols to implement the
     /// graceful shutdown logic.
     ///
-    /// This `shutdown` method is required by implementors of the
+    /// This `shutdown` method is required by implementers of the
```

Comment:
Normally I wouldn't nit this, but since we're going through with spelling/grammar: I think we could make this even better if we said "implementations"

Reply:
This is mostly a passing thought, but the "-or" vs "-er" seems to be a sort of no-mans-land in American English from my cursory Google searches.  I think "implementations" avoids that and it follows more closely to the Rust documentation which refers to "implementations" rather than "implement(o|e)rs".

I'm going to approve it either way but curious on your thoughts!

Reply:
I was following Vim's spell check (which uses `hunspell`'s en_US dictionary). The `-er` "feels" more right, but it does seems to be just more popular. Do you want a PR for `implementations`? Maybe "This `shutdown` method must be provided by implementations…"?